Testing/ORKTest/ORKTest/Resources/Localized/Base.lproj/LaunchScreen.xib

```diff
@@ -13,12 +13,14 @@
             <subviews>
                 <view contentMode="scaleToFill" translatesAutoresizingMaskIntoConstraints="NO" id="KqP-Vs-eBe" userLabel="topBackground">
                     <rect key="frame" x="0.0" y="0.0" width="480" height="42"/>
+                    <animations/>
                     <color key="backgroundColor" white="0.96999999999999997" alpha="1" colorSpace="calibratedWhite"/>
                     <constraints>
                         <constraint firstAttribute="height" constant="42" id="uIJ-4C-6cJ"/>
                     </constraints>
                 </view>
             </subviews>
+            <animations/>
```

Comment:
This is probably because of me using a newer Xcode version (7.1.1). Shouldn't bother anybody...